src/components/Table/exampleData.js

```diff
@@ -0,0 +1,294 @@
+export const data = [
```

Comment:
we should move this file to this folder `library/data` where we already have a `user.js` file and it is set globally in the `library/setup.js` file